caffe2/python/serialized_test/serialized_test_util.py

```diff
@@ -27,8 +27,10 @@
 
 def given(*given_args, **given_kwargs):
     def wrapper(f):
-        hyp_func = hy.seed(0)(hy.settings(max_examples=1)(hy.given(*given_args, **given_kwargs)(f)))
-        fixed_seed_func = hy.seed(0)(hy.settings(max_examples=1)(hy.given(
+        # In fact, 200 is the hy's default value
+        deadline_val = 10000 if os.name == 'nt' else 200
```

Comment:
@malfet, thank your questions. 
According to https://hypothesis.readthedocs.io/en/latest/settings.html
_register_profile_ is used to provide the different default value.
The decorator _given_ and _settings_ is used to tweak the default values
```
    @given(X=hu.tensor(), in_place=st.booleans(),
                  engine=st.sampled_from(["", "CUDNN"]), **mu.gcs)
    @settings(deadline=10000)
    def test_relu(self, X, in_place, engine, gc, dc):
```

For serialized tests, caffe2 provide a new decorator _serial.given_ to replace hypothesis _given_
For example. 
```
    @serial.given(X=hu.tensor(elements=hu.floats(-3.0, 3.0)),
                  n=hu.floats(min_value=0.5, max_value=2.0),
                  in_place=st.booleans(), **hu.gcs)
    def test_relu_n(self, X, n, in_place, gc, dc):
```

In the serial.given, the hy.settings(max_examples=1) changes the registered profile deadline from 10000 to 200. Because the function of hy.setting default value of dealine is 200.

BTW, in serial.given, it calls the hy.settings, so it throws error that setting has been defined. that is, we could write code like 
```
@serial.given(XXX)
@settings(deadline=xxx)
```